tensorflow/core/kernels/image_resizer_state.h

```diff
@@ -27,13 +27,13 @@ limitations under the License.
 #include <algorithm>
 #include <array>
 
-#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
```

Comment:
for some reason, existing code puts this first in a lot of places.  do you know why?

(same on other files, if this ends up getting moved back).